modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/utils/imagePropsTransformer.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
```

Comment:
As the `<CardWithInfo />` component accepts both, a string for the image `src` attribute, and an object with valid image attributes it sounds like a good idea to allow rendering the images coming from Liferay upload component or Documents And Media, that returns an object like.

```
 {
    "id":75639,
    "link":{
        "href":"/documents/75636/75638/mark-duffel-U5y077qrMdI-unsplash.jpg/e447bc81-7a4a-b89a-fb3c-4bf727ae9746?version=1.0&t=1711546759424&download=true&objectDefinitionExternalReferenceCode=8973c6f8-ea54-fd05-d996-815d784e4964&objectEntryExternalReferenceCode=5c2d0dfc-39c6-8f33-869c-4b46f7c997f6",
        "label":"mark-duffel-U5y077qrMdI-unsplash.jpg"
    },
    "name":"mark-duffel-U5y077qrMdI-unsplash.jpg"
}
```

Having it as a helper method will allow its usage for the Table and List component.